sdk/appcenter-analytics/src/main/java/com/microsoft/appcenter/analytics/Analytics.java

```diff
@@ -531,4 +608,12 @@ private synchronized void setInstanceListener(AnalyticsListener listener) {
     WeakReference<Activity> getCurrentActivity() {
         return mCurrentActivity;
     }
+
+    @Override
+    public synchronized void onStarted(@NonNull Context context, String appSecret, String transmissionTargetToken, @NonNull Channel channel) {
+        super.onStarted(context, appSecret, transmissionTargetToken, channel);
+
+        /* Initialize a default transmission target if a token has been provided. */
+        mDefaultTransmissionTarget = getTransmissionTarget(transmissionTargetToken);
```

Comment:
call instance method you are already in instance here, this goes with the other comment to do nothing in static except redirecting to instance.